### PR TITLE
[Wallet] Implement new send & import flow

### LINKED ISSUE #1523: Verification V2 (Optional Verification)
### Expected Behavior

Full Spec: https://docs.google.com/document/d/15oOHJlJk-yk19JRMZLO-Wm-iEB6tfgW4XcTtnNsJuEk/edit?pli=1#

Figma for onboarding: https://www.figma.com/file/9lBnIzsABhO8oXE0d0c8Km/Onboarding-and-Verification?node-id=10%3A557

Figma for tx flow: https://www.figma.com/file/yipmtvjgUreKhZyuK2slFg/Transaction-Flow%3A-Verification-and-Contact-Permissioning?node-id=0%3A1

Includes the Following (feel free to split up):
- invite code optional
- new verification flow
- post-onboarding prompts to re-verify

### Current Behavior

What is the current behavior?

### LINKED ISSUE #1719: Block sending to phone numbers for non-verified users
Subtask, see #1523 for details

### LINKED ISSUE #1722: Defer contact import to Send screen
Subtask, see #1523 for details

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Remove Import Contacts screen from NUX flow
- Implement new designs for Send screen, including sync, verification, and import banners
- Update behavior for send search bar
- Refactor some things between RecipientPicker and Send/Invite
- Fix bug preventing searching recipients by address
- Add settings option to toggle verification state

### Tested

Ran through the NUX and Send flows on Android and iOS

### Related issues

- Fixes #1722 
- Fixes #1719 
- Fixes #1523

### Backwards compatibility

Yes

wa-send-needs-ver
wa-send-perms
wa-send-needs-import
wa-send-sync

### CODE REVIEW

File: packages/mobile/locales/es-419/sendFlow7.json [hunk 1]
```diff
@@ -71,5 +71,20 @@
   "askForContactsPermissionAction": "Accede a tus contactos",
   "newAccountBalance": "Nuevo saldo de cuenta: ",
   "estimatingFee": "Estimando tarifa ",
-  "estimatedFee": "Tarifa estimada: "
+  "estimatedFee": "Tarifa estimada: ",
+  "verificationCta": {
```
Comment (i1skn) [inline packages/mobile/locales/es-419/sendFlow7.json:75]: Can we mark these with `~~`, so we know we need to translate them later.

File: packages/mobile/src/identity/contactMapping.ts [hunk 2]
```diff
@@ -27,40 +30,15 @@ import { contractKit } from 'src/web3/contracts'
 import { getConnectedAccount } from 'src/web3/saga'
 
 const TAG = 'identity/contactMapping'
-const MAPPING_CHUNK_SIZE = 25
-const NUM_PARALLEL_REQUESTS = 3
+const MAPPING_CHUNK_SIZE = 50
```
Comment (i1skn) [inline packages/mobile/src/identity/contactMapping.ts:33]: Just curious on how these were calculated :)

File: packages/react-components/icons/Error.tsx [hunk 3]
```diff
@@ -5,21 +5,23 @@ import Svg, { Circle, Path } from 'svgs'
...
   }
 
   render() {
     return (
-      <View style={this.props.style}>
+      <View style={this.props.style} testID={'ErrorIcon'}>
         <Svg
-          width={'16'}
-          height="16"
+          width={this.props.width}
```
Comment (i1skn) [inline packages/react-components/icons/Error.tsx:23]: 👍

**Review by i1skn — APPROVED** [review]
Great work!! Overall LGTM! Just left minor comments.    

Also, I'm curious which importing format do we want to adopt project-wise?
```
import React, { useEffect } from 'react';
useEffect(...
```
or
```
import * as React from 'react'
React.useEffect(...
```

File: packages/mobile/src/identity/contactMapping.ts (hunk 2 above)
Comment (jmrossy) [reply packages/mobile/src/identity/contactMapping.ts:33]: Through experimentation

File: packages/mobile/locales/es-419/sendFlow7.json (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/locales/es-419/sendFlow7.json:75]: Will fix right now, thanks!

File: packages/mobile/src/identity/contactMapping.ts (hunk 2 above)
Comment (jmrossy) [reply packages/mobile/src/identity/contactMapping.ts:33]: A while ago

**Pull request merged**